pkg/driver/controller.go

```diff
@@ -924,7 +924,7 @@ func newCreateVolumeResponse(disk *cloud.Disk, ctx map[string]string) *csi.Creat
 		}
 	}
```

Comment:
Should we change our e2e tests to use new topology key, as mentioned in this TODO [here](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/ec8718afe87806ed5627c6fec54e54240feb5289/tests/e2e/driver/ebs_csi_driver.go#L54-L55) 
?